src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/VariableMultiConverter.shared.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using Xamarin.CommunityToolkit.Extensions;
+using Xamarin.CommunityToolkit.Helpers;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class VariableMultiConverter : MultiValueConverterExtension, IMultiValueConverter
+	{
+		public MultiBindingCondition ConditionType { get; set; }
+
+		public int Count { get; set; }
+
+		public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+		{
+			if (values == null || !targetType.IsAssignableFrom(typeof(bool)))
+				return false;
+
+			var amount = 0;
```

Comment:
I propose to simplify the code a little.

```csharp

var boolValues = values.OfType<bool>();
if (boolValues.Count() != values.Count())
	return false;

var count = boolValues.Where(v => v).Count();

var amount = ConditionType switch {
	MultiBindingCondition.Any => 1,
	MultiBindingCondition.Count => Count,
	_ => boolValues.Count()
};

return amount == count;
```

Reply:
This won't work entirely, right? Since for `Any` it can also be 2 and be valid. `Count` will be 2, but it will expect 1. I refactored a bit to support that too.

Reply:
Yes, you are right. It would work for ```>=```